docs/en/drivers/ios-uiautomation.md

```diff
@@ -0,0 +1,225 @@
+> with reality, and the driver will be removed in a future version of Appium.
+> To begin iOS automation with Appium today, please use the [XCUITest
+> Driver](/docs/en/drivers/ios-xcuitest.md) instead.
+
+Appium's former method for iOS app automation was based on `UIAutomation`, an
+Apple-provided framework that shipped with the iOS SDK until iOS 10, when it
+was removed. `UIAutomation` was one of the tools included in Apple's
+`instruments` profiling system, and provided a JavaScript API that ran
+synchronously in the context of a single app. The Appium UIAutomation driver
+established an asynchronous, session-based WebDriver front end for this API.
+
+Development of the UIAutomation driver is done at the
+[appium-ios-driver](https://github.com/appium/appium-ios-driver) repo.
+
+### Requirements and Support
+
+(In addition to Appium's [general requirements](#TODO))
+
+* Xcode 7 or lower.
+* iOS simulators or devices with version 9.3 or lower.
+* All versions of Appium ship with this driver.
+* For correct functioning of the driver, see additional setup below.
+
+### Usage
+
+The way to start a session using the UIAutomation driver is to set the
+`platformName` [capability](#TODO) in your [new session request](#TODO) to the
+value of `iOS`. Of course, you must also include appropriate `platformVersion`,
+`deviceName`, and `app` capabilities, at a minimum.
+
+### Capabilities
+
+The UIAutomation driver supports a number of standard [Appium
+capabilities](/docs/en/writing-running-appium/caps.md), but has an additional
+set of capabilities that work for this driver only (see the "iOS" section of
+the aforementioned doc).
+
+To automate Safari instead of your own application, leave the `app` capability
+empty and instead set the `browserName` capability to `Safari`.
+
+### Commands
+
+To see the various commands Appium supports, and specifically for information
+on how the commands map to behaviors for the UIAutomation driver, see the [API
+Reference](#TODO).
+
+### Simulator Setup
+
+1. To allow the iOS simulator to be automated by Instruments, you need to
+   modify the authorization database for the system. Appium provides an easy
+   way to do this by installing and running an authorization script:
+
+    ```
+    npm install -g authorize-ios
+    sudo authorize-ios
+    ```
+
+1. By default, Instruments-based automation is limited by the inclusion of
+   a 1-second hard-coded delay between commands, implemented for obscure
+   reasons by Apple's engineers. There is a way around this limitation called
+   [instruments-without-delay](https://github.com/facebookarchive/instruments-without-delay)
+   (IWD). IWD ships with Appium for Xcode versions < 7. For 7.x and up, IWD
+   must be installed manually by the user in advance of using Appium. The way
+   to do this is as follows:
+
+    * Clone the [appium-instruments](https://github.com/appium/appium-instruments) repository
```

Comment:
This doesn't exist anymore. The script is now in https://github.com/appium/appium-ios-driver/blob/master/bin/xcode-iwd.sh